tracer/src/Datadog.Trace.Tools.Runner/AnalyzeInstrumentationErrorsSettings.cs

```diff
@@ -5,25 +5,23 @@
 
 #nullable enable
 
+using System.ComponentModel;
 using Spectre.Console.Cli;
 
 namespace Datadog.Trace.Tools.Runner
 {
     internal class AnalyzeInstrumentationErrorsSettings : CommandSettings
```

Comment:
Since you've turned all the arguments into optional things, are we giving the user a reasonable error message if they don't specify anything at all?

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1